scripts/lib/authority-and-program-updater.ts

```diff
@@ -1,6 +1,5 @@
 import fs from 'fs';
 import { Project } from 'ts-morph';
-
 const project = new Project({
```

Comment:
Doesn't really matter, but is this a prettier change? Otherwise I'd say better to have a line in between imports and everything else.

Reply:
I'm guessing it was an accidental change - prettier doesn't seem to care either way. Adding back.